### PR TITLE
Squashing

### PR DESCRIPTION
**Pull request by anshuvermaa**

### CODE REVIEW

File: hello.cpp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#include<iostream>
+using namespace std;
+
+int main(){
+<<<<<<< HEAD
```
Comment (pranavgade20) [inline hello.cpp:5]: you should remove theses 'weird' lines that git uses you to tell which version had which changes. 

Remove them and squash the last two commits, and I'll accept this

**Comment by pranavgade20** [comment]
closing for now. please fix https://github.com/pranavgade20/practice-git/pull/93#discussion_r824487420 and reopen the issue.

File: hello.cpp (hunk 1 above)
Comment (anshuvermaa) [reply hello.cpp:5]: which line

Comment (anshuvermaa) [reply hello.cpp:5]: only i have to write  cout<<"hello world "; ?
not part like #include like stuf

Comment (pranavgade20) [reply hello.cpp:5]: remove the lines saying ` <<<<<<< HEAD ` , `========`, and ` >>>>>>> hello-git`

**Pull request closed without merging**